src/cursor/cur_backup.c

```diff
@@ -345,7 +345,7 @@ __backup_add_id(WT_SESSION_IMPL *session, WT_CONFIG_ITEM *cval)
             __wt_verbose(session, WT_VERB_BACKUP, "Free blk[%u] entry", i);
             break;
         }
-        __wt_verbose(session, WT_VERB_BACKUP, "Entry blk[%u] has flags 0x%" PRIx64, i, blk->flags);
+        __wt_verbose(session, WT_VERB_BACKUP, "Entry blk[%u] has flags 0x%" PRIx16, i, blk->flags);
```

Comment:
It would be nice if code like this could be automatically updated if/when the size of the `flags` variable changes. But I'm not seeing a tractable way to do that.

Reply:
WT-9444 should be able to do this.